src/Identity.OnPremisesPublishingProfiles/Identity.OnPremisesPublishingProfiles/Microsoft.Graph.Identity.OnPremisesPublishingProfiles.psd1

```diff
@@ -71,39 +71,39 @@ FormatsToProcess =
 # NestedModules = @()
 
 # Functions to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no functions to export.
-FunctionsToExport = 'Get-MgOnPremisPublishingProfileAgent', 
+FunctionsToExport = 'Get-MgOnPremisPublishingProfile', 
```

Comment:
Should be OnPremise.  It's missing an e

Reply:
This looks like a typo from AutoREST when getting a singular of `onPremises`. I'll open an issue on their end as well.